src/Ng/SearchIndexFromCatalogCollector.cs

```diff
@@ -44,7 +44,7 @@ public SearchIndexFromCatalogCollector(ILogger logger, Uri index, IndexWriter in
             IEnumerable<JObject> catalogItems = await FetchCatalogItems(client, items, cancellationToken);
 
             var numDocs = _indexWriter.NumDocs();
-            _logger.LogInformation(string.Format("Index contains {0} documents.", _indexWriter.NumDocs()));
+            _logger.LogInformation(string.Format("Index contains {0} documents.", numDocs));
```

Comment:
Svetlana, any followup here? i only changed the call to _indexWriter.NumDocs() to the actual value. What should this string actually be?

---
In reply to: [75553808](https://github.com/NuGet/NuGet.Services.Metadata/pull/87#discussion_r75553808) [](ancestors = 75553808)